### PR TITLE
[explorer] Stake button on validator page

### PR DESCRIPTION
**Pull request by Jordan-Mysten**
This adds the stake button to the validator page, flagged by a feature flag disabled in prod for now. I also updated to pull the latest validator metadata.

This also fixes a bug in wallet where the features have not yet been loaded causing a redirect to the token pages when trying to stake.

### CODE REVIEW

**Review by pchrysochoidis — COMMENTED** [review]
Just one thing about the stake view redirect, rather than that lgtm

File: apps/explorer/src/pages/validator/ValidatorDataTypes.ts [hunk 1]
```diff
@@ -142,7 +142,10 @@ export interface NextEpochValidator {
 
 export interface NextEpochValidatorFields {
     consensus_address: number[];
-    name: number[];
+    name: number[] | string;
```
Comment (Jibz1) [inline apps/explorer/src/pages/validator/ValidatorDataTypes.ts:145]: I think we can import validator types from the sdk

**Review by Jibz1 — APPROVED** [review]

File: apps/wallet/src/ui/app/staking/home/index.tsx [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
 import { DelegationDetail } from '../delegation-detail';
 import StakePage from '../stake';
 import { Validators } from '../validators';
+import { FEATURES } from '_src/shared/experimentation/features';
+
 export function Staking() {
+    const navigate = useNavigate();
+    const { source, on } = useFeature(FEATURES.STAKING_ENABLED);
+
+    // Handle the case where features take too long to load, and we'll just navigate home:
```
[the remark this answers is not in the document]
Comment (Jordan-Mysten) [reply apps/wallet/src/ui/app/staking/home/index.tsx:17]: @pchrysochoidis Ah good call. I'll try this out

File: apps/explorer/src/pages/validator/ValidatorDataTypes.ts (hunk 1 above)
Comment (Jordan-Mysten) [reply apps/explorer/src/pages/validator/ValidatorDataTypes.ts:145]: Yeah there's a lot of changes we need to make around this but will punt it until post-testnet

**Pull request merged**